pkgs/development/python-modules/dicom2nifti/default.nix

```diff
@@ -39,24 +39,39 @@ buildPythonPackage rec {
     scipy
   ];
 
+  patches = [
+    # Hotfix for the OverflowError in tests and package usage.
+    # TODO(@osbm): Remove this patch when the upstream is fixed.
+    (fetchpatch {
+      url = "https://github.com/icometrix/dicom2nifti/pull/152.patch";
+      name = "fix-tests.patch";
+      sha256 = "sha256-57t1a55sBasZzR64kUOqKzh+JiUwGWm8xMy8MN6FiaQ=";
+    })
+  ];
+
   postPatch = ''
     substituteInPlace tests/test_generic.py --replace-fail "from common" "from dicom2nifti.common"
     substituteInPlace tests/test_ge.py --replace-fail "import convert_generic" "import dicom2nifti.convert_generic as convert_generic"
   '';
 
   nativeCheckInputs = [
-    pytestCheckHook
+    gdcm
```

Comment:
`gdcm` appears in `nativeCheckInputs` as well as `propagatedBuildInputs`

Reply:
i added that because some tests fail in the absence of the gdcm package in the testing phase